_fixture/certs/README.md

```diff
@@ -0,0 +1,8 @@
+To Generate valid certificate and private key use following command:
```

Comment:
Fix typo => `To generate a valid certificate and private key use the following command`

Reply:
fixe ... with multiple commits. please use squash when merging.